Python/Tests/Core.UI/BasicProjectTests.cs

```diff
@@ -1351,8 +1351,8 @@ public bool X
             CountIs(itemCount, "HelloWorld.py", 0);     // not included because the actual name is Program.py
         }
 
-        [TestMethod, Priority(0)]
-        [HostType("VSTestHost"), TestCategory("Installed")]
+        //[TestMethod, Priority(UITestPriority.P0)]
```

Comment:
Did you consider using [ignore] ?

There is a benefit that we still see the tests in TextExplorer

Reply:
I did, but these tests are defined in the PythonToolsUITests project, as opposed to the PythonToolsUITestsRunner which is where the tests are actually discovered and executed from.